docs/content/en/docs/packages/packagesecrmigration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Packages regional ECR migration"
+linkTitle: "Packages regional ECR migration"
+weight: 4
+description: >
+    Migrating packages ECR registry to packages registry for EKS Anywhere subscription
+---
+
+When an EKS Anywhere Enterprise Subscription is created, the AWS account that created the subscription is granted access to EKS Anywhere Curated Packages in the AWS Region where the subscription was created. If an existing EKS Anywhere cluster has curated packages access configured with the non-subscription based licens; we need to migrate the packages to consume the regional ECR registries.
+
+<details>
+  <summary>Expand for packages registry to AWS Region table</summary>
+  <br /> 
+  {{% content "../clustermgmt/support/packages-registries.md" %}}
+</details>
+
+### Steps for Migration
+1. Ensure you have an active EKS Anywhere Enterprise Subscription. For more information, refer [Purchase subscriptions.]({{< relref "../clustermgmt/support/purchase-subscription.md" >}})
+
+2. If the EKS Anywhere Enterprise Subscription AWS account and the existing IAM user static credentials for curated packages on the cluster are different, we need to update the `aws-secret` object on the cluster with new credentials. Refer [Updating the package credentials 
+.]({{< relref "./packagecontroller.md#updating-the-package-credentials" >}})
+
+3. Edit the `ecr-credential-provider-package` package on the cluster and update `matchImages` to point to the private ECR for the region you have a subscription. Example, `346438352937.dkr.ecr.us-west-2.amazonaws.com` for `us-west-2`.
+    ```bash
+    kubectl edit package ecr-credential-provider-package  -n eksa-packages-<cluster name>
+    ```
+    This causes `ecr-credential-provider-package` pods to rollout and configure’s kubelet to use aws credentials for pulling images from the new ECR registry.
+
+4. Edit the `PackageBundleController` object on the cluster and set the `defaultImageRegistry` and `defaultRegistry` to point to the private ECR for the region we have a subscription.
```

Comment:
Nit: "... to point to the ECR package registry for the AWS Region where you created your subscription."